isb_web/metrics.py

```diff
@@ -56,3 +54,15 @@ def root():
     metrics.solr_counts = solr_counts_by_authority()
     metrics.solr_scrape_duration_seconds = time.time() - db_end_time
     return PlainTextResponse(metrics.metrics_string())
+
```

Comment:
This part is a little goofy -- I couldn't figure out why things were behaving different for `/metrics` vs. `/metrics/` requests.  In the end I threw my hands up and just explicitly handled both of them.  I did look in the docs if there was a way to just configure this so I didn't have to declare separate paths, but I didn't find anything.

Reply:
Yes, that is definitely what was happening, and for whatever reason prometheus was choking on that.  It seemed like an undesirable behavior, so I just made both work with no redirect.